src/components/ScheduleToggle.vue

```diff
@@ -20,11 +20,15 @@ export default {
   },
   computed: {
     ...mapGetters('tenant', ['tenant', 'role']),
+    ...mapGetters('license', ['hasPermissions']),
     archived() {
       return this.flow.archived
     },
     isReadOnlyUser() {
-      return this.role === 'READ_ONLY_USER'
+      return (
+        !this.hasPermissions('create', 'role') &&
+        !this.hasPermissions('create', 'flow')
```

Comment:
I think we need to check 2 things here: some users will have `create:run`  permission but not `delete:run` (and vice versa) so the quick run button will need to be gated on `create:run` and the schedule toggle will need to be gated for both scenarios; when a user has `create:run` but not `delete:run` they should be able to turn a schedule on but not off etc (same comment as below)